spectree/plugins/flask_plugin.py

```diff
@@ -187,11 +188,25 @@ def validate(
             after(request, response, req_validation_error, None)
             abort(response)
 
-        response = make_response(func(*args, **kwargs))
+        result = func(*args, **kwargs)
+
+        if resp and isinstance(result, tuple) and isinstance(result[0], BaseModel):
+            if len(result) > 1:
+                result = result[0].dict(), *result[1:]
+            else:
+                result = (result[0].dict(),)
+
+            skip_validation = True
+        elif resp and isinstance(result, BaseModel):
+            result = result.dict()
+            skip_validation = True
+
+        response = make_response(result)
```

Comment:
```suggestion
        result = func(*args, **kwargs)

        if resp and isinstance(result, tuple) and isinstance(result[0], BaseModel):
            if len(result) > 1:
                model, status, *rest = result
                if isinstance(model, resp.find_model(status)):
                    skip_validation = True
                    result = (model.dict(), status, *rest)

        elif resp and isinstance(result, resp.find_model("200")):
            skip_validation = True
            result = result.dict()

        response = make_response(result)
```

I think we can still check the status code for "flask".

Reply:
Thanks, I've added this